### DISCUSSION TITLE
OPT-070 reserved execution justification/description

### DISCUSSION THREAD

**Discussion by odscjen** [OP-TED/eForms-SDK#254] [Q&A] [answered]
OPT-070 doesn't appear in the SDK guidance pages but from it's xpathAbsolute it appears to be an additional description field to sit with BT-736. So the block would look like:

```
<cac:ContractExecutionRequirement>
    <cbc:ExecutionRequirementCode listName="reserved-execution">not-known</cbc:ExecutionRequirementCode>
    <cbc:Description languageID="ENG">...</cbc:Description>
</cac:ContractExecutionRequirement>
```

with the code being BT-736 and the description being OPT-070. 

Can you provide some guidance as to how this description field will be used? Will it contain new information not covered by another existing term?

**Comment by YvesJo** [comment, marked as answer]
OPT-070 may only be used under some circumstances for the CEI (Call for Expression of Interest) form which is not covered by the eForms regulation.

**Reply by jpmckinney** [reply]
Where can we learn more about the CEI, to better understand the semantics of this field?

**Reply by YvesJo** [reply]
For business related questions, preferably send them to the Helpdesk ted@publications.europa.eu
